Source/Directory.Build.props

```diff
@@ -1,8 +1,8 @@
 <Project>
 
   <PropertyGroup>
-    <VersionPrefix>3.9.1.41027<!--Version 3.9.1, year 2018+4, month 10, day 27.--></VersionPrefix>
+    <VersionPrefix>3.10.0</VersionPrefix>
```

Comment:
Why only partially change this now?

Reply:
I can't fully change it since I don't know the release date yet. I have to bump the version number so I can use the nightly build to develop the VSCode extension against.